tests/test_isrMisc.py

```diff
@@ -88,7 +88,7 @@ def test_linearize(self):
         """Assert that the linearize task does not error when a linearity is requested.
         """
         for linearityTypeName in ('LookupTable', 'Squared', 'Unknown'):
```

Comment:
Do you need to check for the other types of LinearityTypeName's such as polynomial?